cmd/subctl/upgrade.go

```diff
@@ -107,12 +112,9 @@ func upgradeSubctl(status reporter.Interface) (string, error) {
 
 	// If the user hasn't specified a version, try to find the latest release on GitHub
 	if upgradeSubctlVersion == "" {
-		client := github.NewClient(nil)
-		latestRelease, _, err := client.Repositories.GetLatestRelease(context.TODO(), "submariner-io", "releases")
-
-		// If we can't determine the latest release, we'll force a download and delegate to get.submariner.io
-		if err == nil {
-			upgradeSubctlVersion = *latestRelease.TagName
+		tag, err := retrieveLatestReleaseTag()
+		if err != nil {
```

Comment:
We should at least log the error instead of silencing it. Also shouldn't this be:


```suggestion
		if err == nil {
```

Reply:
Oops, yes, it should be! We didn’t log the error in the past, and where should we log it? This is in `subctl`, so apart from the screen there isn’t anywhere to log it. The error is unlikely to be user-actionable so this would probably only confuse people...

Reply:
We can log it to the `status` as a warning. It may not be actionable but it would be informational, in case something else goes wrong as a side-effect that might provide a clue. I don't think it's best practice to ignore errors.